go.sh

```diff
@@ -26,29 +26,35 @@ if [ "${LOCATIONS_CSV}" == "" ]; then
     LOCATIONS_CSV=/
 fi
 
+NAXSI_LISTEN_PORT="10418"
```

Comment:
make this `NAXSI_LISTEN_PORT="${NAXSI_LISTEN_PORT:-10418}"` to make this overridable...?